packages/web-react/src/types/button.ts

```diff
@@ -21,6 +21,8 @@ export interface ButtonBaseProps<C = void, S = void> extends ChildrenProps, Styl
   isDisabled?: boolean;
```

Comment:
Add to the comment above?

```suggestion
  /** Whether the button is disabled or in a loading state. */
  isDisabled?: boolean;
```

Reply:
But this is not correct. You set isLoading when the button is in the loading state. You don't set isDisabled too.